src/go/rpk/pkg/tuners/irq/device_info.go

```diff
@@ -99,6 +100,9 @@ func (deviceInfo *deviceInfo) GetIRQs(
 			}
 		}
 	}
+
+	sort.Ints(irqs)
```

Comment:
This allows the tuner to choose the _same_ affinity each time and also mostly an affinity that is easy to verfiy, i.e. following a pattern like 

001
002
004
008
010
etc

vs random.